src/rom_header_rhh.c

```diff
@@ -0,0 +1,26 @@
+#include "global.h"
+#include "constants/expansion.h"
+
+// Similar to the GF ROM header, this struct allows external programs to
+// detect details about Expansion.
+// For this structure to be useful we have to maintain backwards binary
+// compatibility. This means that we should only ever append data to the
+// end. If there are any structs as members then those structs should
+// not be modified after being introduced.
+struct RHHRomHeader
+{
+    char rhh_magic[4]; // 'PORY'. Useful to locate the header if it shifts.
```

Comment:
I'm thinking we need a longer word, seeing that only 4 characters might give false-positives due to how likely it is to find 4 bytes with that combination.

Reply:
Maybe `'RHHEXP'`?